### DISCUSSION TITLE
Not Just 800-53: OSCAL Outside NIST

### DISCUSSION THREAD

**Discussion by xee5ch** [oscal-club/community#23] [Ideas]
I had reached out to some associates via email because of their interest in OSCAL. Surprise, surprise, OSCAL at its core is developed by NIST, but it is not designed or publicized as exclusive to NIST RMF and 800-53. I have wanted to explore more controls sets that are more industry focused.

The [ISO standards are not openly published _for free_, as covered by social media and the tech press](https://news.ycombinator.com/item?id=26390040). This makes the ISO 27000 series frameworks less than ideal.

Right now, I am considering experiments with [the Cloud Security Alliance's CCM](https://cloudsecurityalliance.org/research/cloud-controls-matrix/).

Open questions for the community:

- Is CSA's CCM interesting? 
- What is a useful control catalog or body of guidance you want to see in OSCAL? Why is interesting? How do you intend to use it?

*NOTE:* For the audience, I am interested in non-NIST stuff. I just _know_ people will say NIST 800-171 Rev2 (or "CMMC controls"; yes they're different!) but humor me and do not just repeat that. Thanks!

**Comment by rgauss** [comment]
Perhaps something around [OWASP](https://owasp.org/)?

A catalog of their [top ten](https://owasp.org/www-project-top-ten/) (versioned by year?) would probably be a good place to start.

Eventually that could include the mappings to a MITRE CWE catalog.

**Reply by xee5ch** [reply]
> Perhaps something around [OWASP](https://owasp.org/)?
> 
> A catalog of their [top ten](https://owasp.org/www-project-top-ten/) (versioned by year?) would probably be a good place to start.

That is an interesting idea. Someone had mentioned this long ago in one of the dev lunches, if I recall, or at least asked about it.

So the OWASP Top 10, [recent versions](https://github.com/OWASP/Top10/tree/5010a4f6c134fcbb617c140878404ace8e0fb224) and those from recent memory,  are a just a list of categorized vulnerability types, with limited guidance for them, maybe not particularly structured. They don't have a lot of the details to fill out an OSCAL catalog, and do do not really have data for assessment rubric beyond the "How to Prevent" fields (note, that's not really about assessment.

Either during or after that meeting, others discuss the OWASP ASVS, that _was and is_ more applicable from an information model context. Seems the work was considered, but not finished. @tohch4 is a dead account now. :disappointed: You think this is worth picking up?

usnistgov/OSCAL#869

OWASP/ASVS#817
 
> Eventually that could include the mappings to a MITRE CWE catalog.

Even with ASVS, CWE and NIST mapping is done where applicable. It's spotty, but it's there!

https://github.com/OWASP/ASVS/blob/27b8b5be1f5af2b1ef23e1912c5b0656caf2a27b/4.0/docs_en/OWASP%20Application%20Security%20Verification%20Standard%204.0.3-en.json#L53-L55

**Comment by xee5ch** [comment]
It has been a while and there was only feedback from @rgauss, but I will start collecting novel catalog/profile/other example data ideas and begin evaluating the creation of them. Feedback welcome here and elswhere!

https://github.com/oscal-club/examples/issues
